routers/embeddings.py

```
@@ -0,0 +1,334 @@
+                    elif record['file_name'].endswith('.json'):
+                        loader = JSONLoader(file_path = file_path, jq_schema='.', text_content=False)
+                    elif record['file_name'].endswith('.md'):
+                        loader = UnstructuredMarkdownLoader(file_path)
+                    elif record['file_name'].endswith('.csv'):
+                        loader = CSVLoader(file_path = file_path)
+                    elif record['file_name'].endswith('.xlsx') or record['file_name'].endswith('.xls'):
+                        loader = UnstructuredExcelLoader(file_path)     
+                    
+                    data.extend(loader.load())
+            
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)